src/components/sticky-summary/sticky-summary.styles.ts

```diff
@@ -0,0 +1,34 @@
+import { SxStyleProp } from 'rebass';
+
+export const boxStyles = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+
+  '> div': {
+    ml: '14px',
+  },
+};
+
+export const boxButtonStyles = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+
+  div: {
+    ml: '19px',
+  },
+};
+
+export default {
+  boxShadow: '0px -1px 0px #E5E5E5, 0px -2px 0px #FFFFFF',
```

Comment:
In Figma, there is no proper way to add a border on one side only of a component. Then you will find sometimes this odd shadow. Here, it is not a shadow but just a border-top of `1px`. Also, don't forget to always use variables for colors